components/export/ui/src/main/resources/PhenoTips/ExportPreferences.xml

```diff
@@ -120,11 +124,19 @@
       #end
       #foreach ($field in $xclass.properties)
         #if (($activeFields.size() == 0 || $activeFields.contains(${field.name})) &amp;&amp; !$ignored.fields.contains(${field.name}) &amp;&amp; !$ignored.types.contains(${field.type}))
-        &lt;li&gt;
-          &lt;label for="${targetFieldName}_${field.name}"&gt;
-          &lt;input type="checkbox" name="${targetFieldName}" id="${targetFieldName}_${field.name}" value="${field.name}" #if($valueList.contains(${field.name}) || $valueListisEmpty) checked="checked" #end /&gt;
-          ${field.translatedPrettyName}
-        &lt;/li&gt;
+          #if ($field.name == "apgar1")
```

Comment:
This is spaghetti code, a custom hack, don't do this. Instead mark both fields as ignored, and add `apgar` as a custom field, like `genes` and `variants`.